lib/ansible/modules/user.py

```diff
@@ -86,12 +86,13 @@
         version_added: "2.0"
     password:
         description:
-            - Optionally set the user's password to this encrypted value.
-            - On macOS systems, this value has to be cleartext. Beware of security issues.
+            - If provided, set the user's password to the provided encrypted hash (Linux) or plain text password (macOS).
+            - B(Linux:) Enter the hashed password as the value.
```

Comment:
it's not just linux, but most POSIXy/unixy systems (i know Mac OS is BSD derived but here it deviates from the norm).

Reply:
Feel free to push to this branch with the wording you like.